code/modules/ninja/ninja_event.dm

```diff
@@ -12,6 +12,7 @@ Contents:
 /datum/round_event_control/ninja
 	name = "Space Ninja"
 	typepath = /datum/round_event/ghost_role/ninja
+	dead_weight_multiplier = DEAD_WEIGHT_LOW
```

Comment:
Ninja is not a good event to scale with low pop, might lead to suffering if this is more frequent since ninja is somewhat powerful(not entirely sure how they are now).